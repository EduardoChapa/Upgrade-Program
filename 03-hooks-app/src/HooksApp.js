import React from 'react';

function HooksApp() {
    return (
        <div>
            This is the main component!
        </div>
    );
}

export default HooksApp;