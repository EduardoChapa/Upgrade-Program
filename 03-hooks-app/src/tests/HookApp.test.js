import React from 'react';
import { shallow } from 'enzyme';
import HooksApp from "../HooksApp";


describe('Testing HookApp', () => {
    test('shloud display correctly', () => {
        const wrapper = shallow(<HooksApp />);
        expect(wrapper).toMatchSnapshot();
    });
});