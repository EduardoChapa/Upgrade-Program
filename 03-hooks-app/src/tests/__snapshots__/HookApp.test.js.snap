// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HookApp shloud display correctly 1`] = `
<div>
  This is the main component!
</div>
`;
